lib/builders/JSBuilder.ts

```diff
@@ -13,6 +14,10 @@ export class JSBuilder {
     let mainModule = this._buildMainModule(data.dependencies._mainModule);
     let raw = data.files.js.map(path => fse.readFileSync(path).toString()).join('\n');
```

Comment:
`path` here now causes a lint error:
```js
ERROR: /home/travis/build/angular/material-tools/lib/builders/JSBuilder.ts[15, 33]: Shadowed name: 'path'
```
Can you please rename this `path` variable to `file`?